controllers/volumereplicationgroup_controller.go

```diff
@@ -273,7 +264,15 @@ type VRGInstance struct {
 }
 
 const (
-	vrgFinalizerName = "volumereplicationgroups.ramendr.openshift.io/protection"
+	// Finalizers
+	vrgFinalizerName        = "volumereplicationgroups.ramendr.openshift.io/vrg-protection"
+	pvcVRFinalizerProtected = "volumereplicationgroups.ramendr.openshift.io/pvc-vr-protection"
```

Comment:
(nit) I may have suggested the Name suffix during code review, where the editor doesn't show it is a string and when I wasn't sure if it was function the first time I saw it. I take back that suggestion in favor of making the two names consistent.